packages/web-client/pages/livestreams/new/CreateLivestream.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import { Input, TagsInput, Button } from '@skillfuze/ui-components';
+import withAuth from '../../../utils/withAuth/withAuth';
+import Layout from '../../../components/Layout';
+import LivestreamService from '../../../services/livestreams.service';
+import config from '../../../config/index';
+
+const CreateLiveStream = () => {
+  const livestreamService = new LivestreamService();
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [thumbnailURL, setThumbnailURL] = useState('');
+  const [tags, setTags] = useState([]);
+  const [streamKey, setStreamKey] = useState('');
+  const [error, setError] = useState<any>({});
+  const serverURL = config.streamingServerURL;
+  const handleSubmit = async (event): Promise<void> => {
+    event.preventDefault();
+    try {
+      const stream = await livestreamService.create({ title, description, thumbnailURL });
+      setStreamKey(stream.streamKey);
+    } catch (err) {
+      setError(err);
+    }
+  };
+  return (
+    <Layout>
+      <div className="flex flex-grow px-40 items-center justify-center h-full">
+        <div className="flex flex-col items-center w-1/2">
+          <div className="flex flex-col w-2/3">
+            <h1 className="text-2xl font-bold text-left">New Livestream</h1>
+            <Input className="mt-8 w-full" value={title} type="text" placeholder="Title" onChange={setTitle} />
+            <Input
+              error={error.title}
```

Comment:
The error.title is put on the description field.
Also please add the errors to the other fields, so if we want to add other validations we won't need to change this view again. (thumbnailURL validation for url will be added).